src/modules/command-handler/index.js

```diff
@@ -0,0 +1,257 @@
+const Command = require('../command')
+const Await = require('../await')
+
+/**
+ * A class reprsenting the command handler.
+ */
+class CommandHandler {
+  /**
+   * Create a CommandHandler
+   * @param {String}            prefix        The prefix of commands.
+   * @param {Client}            client        The Eris client.
+   * @param {String}            ownerId       The ID of the bot owner.
+   * @param {QueryBuilder}      knex          The simple-knex query builder.
+   * @param {Command[]|Command} [commands=[]] List of commands to load initially.
+   */
+  constructor (prefix, client, ownerId, knex, commands = []) {
+    /**
+     * The prefix of commands.
+     * @type {String}
+     */
+    this._prefix = prefix
+    /**
+     * The Eris Client.
+     * @type {Client}
+     */
+    this._client = client
+    /**
+     * The simple-knex query builder.
+     * @type {QueryBuilder}
+     */
+    this._knex = knex
+    /**
+     * Array of commands.
+     * @type {Map<String, Command>}
+     */
+    this._commands = new Map()
+    /**
+     * An object containing message data used to wait for user response.
+     * @type {Map<String, AwaitData>}
+     */
+    this._awaits = new Map()
+    /**
+     * The ID of the bot owner.
+     * @type {String}
+     */
+    this._ownerId = ownerId
+    /**
+     * Map of replacers.
+     * @type {Map<String, Replacer>}
+     */
+    this._replacers = new Map([
+      ['LAST', {
+        key: 'LAST',
```

Comment:
It's just how maps work,

```
id => { id: '123' }
```
Is a pretty standard system, it just looks weird because I'm using the constructor here which requires an array of `[id, val]`

Reply:
So why not just have the id? Why do we need the property?